### DISCUSSION TITLE
force-peaceful-mode的行为是什么？

### DISCUSSION THREAD

**Discussion by sectlying** [LeavesMC/Leaves#307] [Q&A] [answered]
请问一下打开force-peaceful-mode后服务器的具体行为，以及开启这个与开启count-all-mobs-for-spawning、per-player-mobs-spawns的关系是什么？
开启per-player-mobs-spawns的条件下：
什么情况下可以使用坚守者伪和平（paper默认不统计这个）？什么情况下可以使用村民伪和平？

**Comment by Bluemangoo** [comment]
由一个比较激进的方式重新计算是否达到伪和平标准。

paper config改法详见
- #265

**Reply by sectlying** [reply]
这个“重新计算是否达到伪和平标准”的怪物是否包括坚守者？
计算是指扫描统计世界的怪物数目是否大于怪物上限吗？
计算达到伪和平标准之后就不再刷怪了吗？

**Reply by Bluemangoo** [reply]
对。

**Comment by s-yh-china** [comment, marked as answer]
是强制计算目标世界的部分实体，无论其属于什么状态，应该是包括监守者的，如果我没记错
对的，到达上限后就不再刷怪
你可以使用/leaves peaceful来查看具体信息



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月12日(周一) 下午2:06
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: Re: [LeavesMC/Leaves] force-peaceful-mode的行为是什么？ (Discussion #307)




 
这个“重新计算是否达到伪和平标准”的怪物是否包括坚守者？
 计算是指扫描统计世界的怪物数目是否大于怪物上限吗？
 计算达到伪和平标准之后就不再刷怪了吗？

**Reply by Bluemangoo** [reply]
你用instanceof Mob的。

**Reply by sectlying** [reply]
好的，知道了，感谢
